src/Row.ts

```diff
@@ -37,15 +37,44 @@ class Row extends RowBase<RowProperties> {
 				classes: this.classes(tableCss.table, css.rowTable)
 			}, [
 				v('tr', columns.map((column) => {
-					const { id, field } = column;
+					const { field, id } = column;
+
+					let value: any;
```

Comment:
Perhaps would be nicer it could be simplified a bit (does assume consolidating the two render functions):

```ts
let value = item.data[ field || id] ? String(item.data[ field || id ]) : '';
if (column.get) {
    value = column.get(item, column);
}

let content: DNode = value;
if (column.render) {
    content = column.render(item, column, value);
}
```